github/table_github_repository_sbom.go

```diff
@@ -0,0 +1,103 @@
+package github
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableGitHubRepositorySbom() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_repository_sbom",
+		Description: "SBOM from a repository.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "repository_full_name",
+					Require: plugin.Required,
+				},
+			},
+			ShouldIgnoreError: isNotFoundError([]string{"404", "403"}),
+			Hydrate:           tableGitHubRepositorySbomList,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "repository_full_name",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("repository_full_name"),
+				Description: "The full name of the repository (login/repo-name).",
+			},
+			{
+				Name:        "spdx_id",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SPDXID"),
+				Description: "The SPDX identifier for the SPDX document.",
+			},
+			{
+				Name:        "spdx_version",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SPDXVersion"),
+				Description: "The version of the SPDX specification that this document conforms to.",
+			},
+			{
+				Name:        "creation_info",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("CreationInfo"),
+				Description: "The version of the SPDX specification that this document conforms to.",
+			},
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+				Description: "The name of the SPDX document.",
+			},
+			{
+				Name:        "data_license",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DataLicense"),
+				Description: "The license under which the SPDX document is licensed.",
+			},
+			{
+				Name:        "document_describes",
+				Type:        proto.ColumnType_STRING,
```

Comment:
For this column, do we get get always a single string, or can we get back a list? Looking at https://docs.github.com/en/rest/dependency-graph/sboms?apiVersion=2022-11-28, the examples show the data as lists.